src/config/keybind.c

```diff
@@ -120,6 +120,7 @@ keybind_create(const char *keybind)
 {
 	xkb_keysym_t sym;
 	struct keybind *k = znew(*k);
+	k->locked = false;
```

Comment:
This isn't needed, `znew()` will zero out all memory which means it default to `false` anyway.